src/pages/starter-kit/checkout/observability.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Observability
+description: Monitor and trace your checkout application with built-in observability tools
+keywords:
+  - Starter Kit
+  - Extensibility
+  - App Builder
+---
+
+# Observability
+
+Monitor and trace your checkout application with built-in observability tools.
+
+## OpenTelemetry Integration
+
+This starter kit includes built-in observability using `@adobe/aio-lib-telemetry`, which provides OpenTelemetry instrumentation for App Builder actions.
+
+### Overview
+
+OpenTelemetry enables you to monitor and trace your checkout application's performance, helping you identify bottlenecks, debug issues, and understand how your actions interact with Adobe Commerce. The integration automatically collects metrics and traces for key checkout operations without requiring manual instrumentation.
+
+### What's Included
+
+The following checkout actions are instrumented with telemetry:
+
+- **collect-taxes**: Tax calculation requests
+- **filter-payment**: Payment method filtering
+- **shipping-methods**: Shipping method calculations
+- **validate-payment**: Payment validation
+
+### Configuration
+
+Telemetry is an optional feature controlled via the `ENABLE_TELEMETRY` environment variable in `app.config.yaml`. It's enabled by default for all checkout actions:
+
+```yaml
+actions:
+  collect-taxes:
+    inputs:
+      ENABLE_TELEMETRY: true
+```
+
+The telemetry behavior and settings are configured in the `actions/telemetry.js` file, which provides centralized control over instrumentation, diagnostics, and export options.
+
+#### Disabling Telemetry
+
+To disable telemetry for the entire application or specific actions, set `ENABLE_TELEMETRY: false` in `app.config.yaml`:
+
+```yaml
+actions:
+  collect-taxes:
+    inputs:
+      ENABLE_TELEMETRY: false
+```
+
+#### Viewing Telemetry Data
```

Comment:
Overall, my personal frustration was, "So, how can we forward logs to Cloud? Where should I make changes?"
In terms of CSK, it would be helpful to know where to modify the code in the project and which documentation to refer to for that:
https://magento.slack.com/archives/C02NVJGJJK1/p1763478116969269?thread_ts=1763390233.298679&cid=C02NVJGJJK1